api/node/index.ts

```diff
@@ -325,6 +341,243 @@ export class ArrayModel<T> extends Model<T> {
     }
 }
 
+/**
+ * Provides rows that are generated by a map function based on the rows of another Model.
+ *
+ * @template T the type of the source model's items
+ * @template U the type of the mapped items
+ *
+ * ## Example
+ *
+ *  Here we have a {@link ArrayModel} holding rows of a custom interface `Name`.
```

Comment:
Please add a sentence here that describes in words what the example map model does, so that when reading code the reader can "verify". For example that the model combines the first and last name fields into a single string with a comma.

Reply:
done